### PR TITLE
[connectedk8s] Update CSP to latest version

### PR DESCRIPTION
**Pull request by sirireddy12**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
connectedk8s

**Comment by wangzelin007** [comment]
Hi [sirireddy12](https://github.com/sirireddy12),
Do you think it is necessary to rerun live test for bump version ？

**Comment by sirireddy12** [comment]
> Hi [sirireddy12](https://github.com/sirireddy12), Do you think it is necessary to rerun live test for bump version ？

H

> Hi [sirireddy12](https://github.com/sirireddy12), Do you think it is necessary to rerun live test for bump version ？

Hi @wangzelin007 , the test cases to test this bump version are different and not included in livetest. I have done the testing manually on my side. We have a tracking item to include more test cases in livetest. Having said that, these test cases include testing our extension on CLI core version <2.30 as well. It is yet to be figured out how to include those cases in livetests sicne azdev framework uses only latest CLI core version

File: src/connectedk8s/HISTORY.rst [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 Release History
 ===============
+1.2.8
+++++++
+
+* Bumping up CSP version to 1.3.019103, bumpup pycryptodome to 3.14.1 support Python 3.10
```
Comment (jiasli) [inline src/connectedk8s/HISTORY.rst:8]:
```suggestion
* Bump up CSP version to 1.3.019103, bump up `pycryptodome` to 3.14.1 support Python 3.10
```

**Review by jiasli — APPROVED** [review]

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**